contracts/token/ERC721/ERC721Token.sol

```diff
@@ -106,6 +106,35 @@ contract ERC721Token is SupportsInterfaceWithLookup, ERC721BasicToken, ERC721 {
     return ownedTokens[_owner][_index];
   }
 
+  /**
+   * @dev Transfers the ownership of a given token ID to another address
+   * Usage of this method is discouraged, use `safeTransferFrom` whenever possible
+   * Requires the msg sender to be the owner, approved, or operator
+   * @param _from current owner of the token
+   * @param _to address to receive the ownership of the given token ID
+   * @param _tokenId uint256 ID of the token to be transferred
+  */
+  function transferFrom(
+    address _from,
+    address _to,
+    uint256 _tokenId
+  )
+    public
+  {
+    super.transferFrom(_from, _to, _tokenId);
+
+    uint256 removeTokenIndex = ownedTokensIndex[_tokenId];
+    uint256 lastTokenIndex = ownedTokens[_from].length.sub(1);
+    uint256 lastToken = ownedTokens[_from][lastTokenIndex];
+
+    ownedTokens[_from][removeTokenIndex] = lastToken;
+    ownedTokens[_from][lastTokenIndex] = 0;
```

Comment:
As per https://github.com/OpenZeppelin/openzeppelin-solidity/issues/977 this line should be safe to remove and save a bit more gas.